models/db.py

```diff
@@ -184,7 +184,37 @@ def sanitize_fields(form):
     for site in current.SITES:
         site_handle = site.lower() + "_handle"
         form.vars[site_handle] = form.vars[site_handle].lower()
-
+#-----------------------------------------------------------------------------
+def inside_institute(form):
+        # Send mail to all users form the same college when someone registers
+    to = []
+    for mail in db(utable.email == form.vars.institute).select():
+        to.append(mail)
+    subject = "New user registered from your college"
+    message = """
+Name: %s %s
+Email: %s
+Institute: %s
+StopStalk handle: %s
+Referrer: %s
+Codechef handle: %s
+Codeforces handle: %s
+Spoj handle: %s
+HackerEarth handle: %s
+HackerRank handle: %s
+              """ % (form.vars.first_name,
+                     form.vars.last_name,
+                     form.vars.email,
+                     form.vars.institute,
+                     form.vars.stopstalk_handle,
+                     form.vars.referrer,
+                     form.vars.codechef_handle,
+                     form.vars.codeforces_handle,
+                     form.vars.spoj_handle,
+                     form.vars.hackerearth_handle,
+                     form.vars.hackerrank_handle)
+    send_mail(to=to, subject=subject, message=message)
```

Comment:
send_mail function does not take a list of `to`'s.
So put every thing in the above loop (`for row in rows`)
and call send_mail in every iteration with a seperate email id.
O wait - move subject variable and message outside the loop and just change the `to`'s and then call send_mail in every iteration with a different `to` and same subject and message